backend/app/assets/javascripts/spree/backend/views/tables/return_items.js

```diff
@@ -0,0 +1,9 @@
+Spree.Views.Tables.ReturnItems = Backbone.View.extend({
+  initialize: function() {
+    if(this.$el.hasClass('return-items-table')) {
+      var tfoot = document.createElement('tfoot')
+      new Spree.Views.Tables.SelectableTable.SumReturnItemAmount({ el: tfoot, model: this.model});
```

Comment:
Please use the same spacing rule for starting and ending inside `{}`

Reply:
Done!